layers/+lang/ruby/README.org

```diff
@@ -48,14 +52,25 @@ This layer supports two different Ruby modes: Emacs’s built-in Ruby Mode and
      '((ruby :variables ruby-enable-enh-ruby-mode t)))
 #+END_SRC
 
-** Prerequisites
+** Backends
+*** Robe
+    The default Ruby-mode backend. See [[Prerequisites]] for the necessary Gems to support Robe.
+*** Language Server Protocal
+    IDE-like backend for ruby. See [[Prerequisites]] for the necessary Gems to support LSP. LSP also supports the DAP debugger which will be 
+
+    automatically configured upon using the LSP backend.
+    Dap-mode can be setup here: https://github.com/emacs-lsp/dap-mode#ruby
+    The
```

Comment:
- Please fix the indentation here, text under headlines should be uniformly formatted like it is done for the prerequisites headline. 
- Also you should add linebreaks to match a line limit of 80 chars in the documentation.
- Please also hide the external link in an org link expression.
- It looks like your text just stops at your last sentence, starting with `The`.

Reply:
Hope you had a good vacation! :)

I'm kind of an org-mode noob. I've tried to make the requested changes, but please let me know if they look ok. Thanks for the feedback!